src/merkle-funder.ts

```diff
@@ -49,53 +63,64 @@ export const fundChainRecipients = async (
 
     // Get block number to use as argument when fetching the transaction count
     if (!getBlockNumberSuccess) {
-      console.log('Failded to fetch block number:', decodeRevertString(getBlockNumberReturndata), chainId);
+      logger.info(`Failded to fetch block number: ${decodeRevertString(getBlockNumberReturndata)}`, logOptions);
```

Comment:
```suggestion
      logger.info(`Failed to fetch block number: ${decodeRevertString(getBlockNumberReturndata)}`, logOptions);
```
Probably also `data`->`Data` but I try not to nit 😆